examples/with-emotion-vanilla/pages/_document.js

```diff
@@ -0,0 +1,34 @@
+import Document, { Html, Head, Main, NextScript } from 'next/document'
+import * as React from 'react'
+import { renderStatic } from '../shared/renderer'
+export default class AppDocument extends Document {
+  static async getInitialProps(ctx) {
+    const page = await ctx.renderPage()
+    const { css, ids } = await renderStatic(() => page.html)
+    const initialProps = await Document.getInitialProps(ctx)
+    return {
+      ...initialProps,
+      styles: (
+        <React.Fragment>
+          {initialProps.styles}
+          <style
+            data-emotion={`css ${ids.join(' ')}`}
+            dangerouslySetInnerHTML={{ __html: css }}
+          />
+        </React.Fragment>
+      ),
+    }
+  }
+
+  render() {
+    return (
+      <Html lang="en">
```

Comment:
```suggestion
      <Html>
```

Lang is automatically controlled by the i18n integration.